src/theme.ts

```diff
@@ -111,6 +111,9 @@ export const windowBreakWidths = {
   breakXL: 1536,
 } as const;
 
+// key for caching user color preference in localStorage
+export const storageColorPreferenceKey = 'colorMode';
```

Comment:
camel case is typically used for mutable variables. Which, I'm aware, is not consistently followed throughout our repo. Regardless, with a global constant I would prefer different naming such as `STORAGE_COLOR_PREFERENCE_KEY` or `COLOR_PREFERENCE_KEY`